docs/docsource/installation/install/index.rst

```diff
@@ -11,4 +11,5 @@ configured, you must install Helm, then use it to install |product|.
 
    install_helm
    install_zenko
+   installing_xdm
```

Comment:
I am concerned about parallelism here. I recommend changing the file name and this link to "install_xdm" as with Helm and Zenko, above.